webxrlayers-1.bs

```diff
@@ -1664,7 +1664,8 @@ This module replaces the steps given by "[=update the pending layers state=]" fr
 <dfn dfn-for="layers">update the pending layers state</dfn> with {{XRSession}} |session| and {{XRRenderStateInit}} |newState|, it must run the following steps:
   1. If |newState|'s {{XRRenderStateInit/baseLayer}} is set:
     1. If the |session| was created with "[=feature descriptor/layers=]" enabled, throw an {{NotSupportedError}} and abort these steps.
-    1. If |newState|'s {{XRRenderStateInit/layers}} is not empty, throw a {{TypeError}} and abort these steps.
+    1. If |newState|'s {{XRRenderStateInit/layers}} is not <code>null</code>, throw a {{NotSupportedError}}.
+    1. Abort these steps.
```

Comment:
This should just be "and abort these steps". If you want to make a nested list, the previous thing needs to end with "perform the following steps" or something, but a single-step sublist is not really common

Reply:
I reflowed the algorithm. I think it's a bit cleaner this way